--- a.boost_1_46_1/libs/thread/src/win32/tss_pe.cpp	2013-03-28 11:24:06 -0400
+++ b.boost_1_46_1/libs/thread/src/win32/tss_pe.cpp	2013-03-28 11:24:45 -0400
@@ -58,6 +58,7 @@
     PIMAGE_TLS_CALLBACK __crt_xl_start__ __attribute__ ((section(".CRT$XLA"))) = 0;
     PIMAGE_TLS_CALLBACK __crt_xl_end__ __attribute__ ((section(".CRT$XLZ"))) = 0;
 }
+#if 0
 extern "C" const IMAGE_TLS_DIRECTORY32 _tls_used __attribute__ ((section(".rdata$T"))) =
 {
         (DWORD) &__tls_start__,
@@ -68,6 +69,7 @@
         (DWORD) 0
 };
 #endif
+#endif


 #elif  defined(_MSC_VER) && !defined(UNDER_CE)
