--- gaul-devel-0.1849-0/util/gaul/gaul_config.h 2005-04-20 12:09:49 -0400
+++ gaul-devel-0.1849-0a/util/gaul/gaul_config.h        2013-03-11 13:47:33 -040
@@ -53,9 +53,9 @@

 #ifdef WIN32
 #  ifdef BUILDING_DLL
-#    define FUNCPROTO __declspec (dllexport)
+#    define FUNCPROTO
 #  else
-#    define FUNCPROTO __declspec (dllimport)
+#    define FUNCPROTO
 #  endif
 #else
 #  define FUNCPROTO

--- gaul-devel-0.1849-0/util/gaul/gaul_config_win.h	2005-04-14 11:08:56 -0400
+++ gaul-devel-0.1849-0a/util/gaul/gaul_config_win.h	2013-03-11 13:48:12 -0400
@@ -39,9 +39,9 @@
  **********************************************************************/

 #ifdef BUILDING_DLL
-#  define FUNCPROTO __declspec (dllexport)
+#  define FUNCPROTO
 #else
-#  define FUNCPROTO __declspec (dllimport)
+#  define FUNCPROTO
 #endif

 #  if defined __CYGWIN32__ && !defined __CYGWIN__
